tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -337,6 +337,12 @@ public function updateShare(string $user, TableNode  $body, string $permissionID
 					"type" => $bodyRows['permissionsRole']
 				]
 			];
+		} elseif (\array_key_exists('role', $bodyRows)) {
+			$body = [
+				"roles" => [
+					GraphHelper::getPermissionsRoleIdByName($bodyRows['role'])
+				]
+			];
```

Comment:
```suggestion

```
Don't mixup the logic for link share and normal resource share into one function.